lavamoat/browserify/main/policy.json

```diff
@@ -4273,22 +4273,6 @@
         "TextEncoder": true
       }
     },
-    "eth-lattice-keyring>gridplus-sdk>secp256k1": {
```

Comment:
this change indicates there's now one less version of the package. 
This adds up with what we see in the lockfile too.
I can confirm you've successfully upgraded secp256k1 to 4.0.4 only.